tfgrid3/mastodon/docker/docker-compose.yaml

```diff
@@ -56,7 +55,7 @@ services:
 
   web:
     # build: .
-    image: tootsuite/mastodon:v4.0.0rc1
+    image: "mahmoudemmad/mastodon:tfconnect"
```

Comment:
please push this image to threefolddev account and use it

Reply:
Updated